src/state/slices/opportunitiesSlice/resolvers/thorchainLending/utils.ts

```diff
@@ -0,0 +1,153 @@
+import type { AccountId, AssetId } from '@shapeshiftoss/caip'
+import type { Result } from '@sniptt/monads'
+import { Err, Ok } from '@sniptt/monads'
+import axios from 'axios'
+import { getConfig } from 'config'
+import type { BigNumber } from 'lib/bignumber/bignumber'
+import { assetIdToPoolAssetId } from 'lib/swapper/swappers/ThorchainSwapper/utils/poolAssetHelpers/poolAssetHelpers'
+import { selectAssetById } from 'state/slices/selectors'
+import { store } from 'state/store'
+
+import { getAccountAddresses, toThorBaseUnit } from '../thorchainsavers/utils'
+import type {
+  Borrower,
+  BorrowersResponse,
+  BorrowersResponseSuccess,
+  LendingDepositQuoteResponse,
+  LendingWithdrawQuoteResponse,
+} from './types'
+
+export const getMaybeThorchainLendingOpenQuote = async ({
+  collateralAssetId,
+  collateralAmountCryptoBaseUnit,
+  receiveAssetId,
+  receiveAssetAddress,
+}: {
+  collateralAssetId: AssetId
+  collateralAmountCryptoBaseUnit: BigNumber.Value | null | undefined
```

Comment:
reason for `| null | undefined`? should be required

Reply:
Same as savers, just a convention for these utils since we `bnOrZero()` it!